### PR TITLE
[multi-asic][cli][chassis-db] Avoid connecting to chassis db for cli commands executed from linecard

### PR DESCRIPTION
**Pull request by vganesan-nokia**
#### What I did

Currently, for all the cli commands, we connect to all databases mentioned in the database_config.json. The database_config.json also includes the databases from chassis redis server from supervisor card. It is unneccessary to connect to databases from chassis redis server when cli commands are executed form linecard. But we need to allow connection to chassis databases when the cli commands are executed from supervisor card. This problem is fixed by this PR

#### How I did it

The constructor of Db() class which is instantiated for every CLI command execution is changed to skip chassis databases from the list of collected databases if the card is not supervisor card. This PR is dependent on sonic-buildimage PR https://github.com/Azure/sonic-buildimage/pull/8065.

#### How to verify it
In voq chassis execute some CLI commands (show, config, etc.) and make sure that commands are executed without any CLI errors no matter whether there is supervisor card or not.

#### Previous command output (if the output of a command-line utility has changed)

No command syntax or outputs changed

#### New command output (if the output of a command-line utility has changed)
N/A

### CODE REVIEW

**Comment by anshuv-mfst** [comment]
Hi @mlorrillere - could you please review, thanks.

**Review by mlorrillere — DISMISSED** [review]

File: utilities_common/db.py [hunk 1]
```diff
@@ -11,7 +11,17 @@ def __init__(self):
         self.cfgdb = ConfigDBConnector()
         self.cfgdb.connect()
         self.db = SonicV2Connector(host="127.0.0.1")
-        for db_id in self.db.get_db_list():
+
+        # Skip connecting to chassis databases in line cards
+        db_list = list(self.db.get_db_list())
+        if not device_info.is_supervisor():
+            try:
+                db_list.remove('CHASSIS_APP_DB')
```
Comment (judyjoseph) [inline utilities_common/db.py:19]: Similar comment, we can have a function in py_common/multi_asic.py or py_common/deviceinfo to get the list of chassis db instances and use here. Will help in future when we add more DBs

**Comment by vganesan-nokia** [comment]
@judyjoseph, would you please re-start the azure pipeline again? The build is still not picking up the dependednt changes from Azure/sonic-buildimage#8065. Thanks

**Pull request merged**